package.json

```diff
@@ -176,17 +176,23 @@
   "scripts": {
     "dev": "concurrently --kill-others -p name -n 'backend,frontend,docs' -c 'blue,green,yellow' 'lein run' 'yarn build-hot' 'yarn docs'",
     "lint": "yarn lint-eslint && yarn lint-prettier && yarn lint-flow && yarn lint-docs-links",
-    "lint-eslint": "yarn && eslint --ext .js --ext .jsx --rulesdir frontend/lint/eslint-rules --max-warnings 0 frontend/src frontend/test",
-    "lint-prettier": "yarn && prettier -l 'frontend/**/*.{js,jsx,css}' || (echo '\nThese files are not formatted correctly. Did you forget to \"yarn prettier\"?' && false)",
+    "lint-eslint": "yarn && eslint --ext .js --ext .jsx --rulesdir frontend/lint/eslint-rules --max-warnings 0 enterprise/frontend/src frontend/src enterprise/frontend/test frontend/test",
+    "lint-prettier": "yarn && prettier -l '{enterprise/,}frontend/**/*.{js,jsx,css}' || (echo '\nThese files are not formatted correctly. Did you forget to \"yarn prettier\"?' && false)",
     "lint-flow": "yarn && flow check",
     "lint-docs-links": "yarn && ./bin/verify-doc-links",
     "lint-yaml": "yamllint",
-    "test": "yarn test-unit && yarn test-timezones && yarn test-integration && yarn test-cypress",
+    "test": "yarn test-unit && yarn test-timezones && yarn test-integration && yarn test-cypress && yarn test-e2e && yarn test-karma",
```

Comment:
This line should be fixed. It reintroduces e2e and karma tests which we removed completely.

Reply:
whoops! good catch